utils/node_local_test.py

```diff
@@ -2081,10 +2081,13 @@ def test_oclass(self):
         """Test container object class options"""
         container = create_cont(self.conf, self.pool, ctype="POSIX", label='oclass_test',
                                 oclass='S1', dir_oclass='S2', file_oclass='S4')
-        run_daos_cmd(self.conf,
-                     ['container', 'query',
-                      self.pool.id(), container.id()],
-                     show_stdout=True)
+        rc = run_daos_cmd(self.conf,
+                          ['container', 'query',
+                           self.pool.id(), container.id()],
+                          show_stdout=True, use_json=True)
+        print(rc)
```

Comment:
`run_daos_cmd` will print stdout anyway so this isn't needed although isn't a problem.  Output is at https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-14130/1/testReport/NLT/test/Unit_Tests___NLT_on_EL_8_8___test_oclass/